### DISCUSSION TITLE
Tactical RMM on existing meshcentral server

### DISCUSSION THREAD

**Discussion by aftechro** [amidaware/tacticalrmm#1746] [General]
Using meshcentral for a while now, is there any way to install the tacticalrmm on existing meshcentral server?
if so, whats the procedure? as well, would recognize existing groups and computers or you recommend to have tacticalrmm installed on fresh new vm?

would be pity to lose existing config of meshcentral, and have to install all new agents on same machines.

Thank you!

**Comment by dinger1986** [comment]
Hello, have you looked at this? https://docs.tacticalrmm.com/mesh_integration/#running-your-own-existing-or-separate-meshcentral-server

**Reply by aftechro** [reply]
seen this, but does not say if this will keep existing groups and computes or if messes with the config. and i dont want the scenario, adding tactical rmm and only devices after tactical installation will show in there and the old devices i have to use meshcentral.

did anyone had similar scenario?

**Reply by dinger1986** [reply]
When I moved to trmm from mesh I used scripts to just swap everything across.

Also you talk about "config" which makes me thing you have customised mesh a fair bit, this might cause issues with trmm so probably best migrating
